### ISSUE TITLE
generateSidebar return type

### ISSUE THREAD

**Issue by reisir** [jooy2/vitepress-sidebar#119]
**Describe the bug** 

```
Type 'Sidebar' is not assignable to type 'Sidebar | undefined'.
  Type 'SidebarMulti' is not assignable to type 'Sidebar | undefined'.
    Type 'SidebarMulti' is missing the following properties from type 'SidebarItem[]': length, pop, push, concat, and 28 more.ts(2322)
```

**To Reproduce**

```js
    sidebar: generateSidebar({}),
```

**Expected behavior**

`generateSidebar()` should not be able to return undefined

**Screenshots** 

![image](https://github.com/jooy2/vitepress-sidebar/assets/93496808/7435d085-b340-4b25-a7fd-dc0135424c06)

**Desktop (please complete the following information):**

- OS: Win 10
- Browser: N/A
- IDE: VS Code

**Additional context**

Everything still works but TypeScript complains. It's annoying and makes the inexperienced user think they're doing something wrong.

Changing `SidebarMultiItem.base` to be required fixes it, idk TypeScript though so might be nonsense.

```diff
-base?: string;
+base: string;
```

**Comment by jooy2** [comment]
Hello, thank you for using vitepress-sidebar. 
This issue has been fixed in `1.18.6`. Can you try it after updating the plugin?

Regards,

**Comment by reisir** [comment]
Yeah after updating the plugin everything works. Thank you for the quick fix.
